### ISSUE TITLE
BBApplication does not get initialized

### ISSUE THREAD

**Issue by RasmusFredensborg** [mustachedk/android-beacon-bacon-lib#2]
If you have your own class that extends Application, which is quite common, the BBApplication class never gets initialized. This is because it is only possible to have one Application class (unless you start extending your own Application class, which gets messy quite quickly). I can see that the only place an Application context is needed, is in the GetFloorImageAsync.java: 
```
WindowManager windowManager = (WindowManager) BBApplication.getContext().getSystemService(Context.WINDOW_SERVICE);
```
We need a way pass down our own Application context or maybe find a way to avoid using the WindowManager all together.

**Comment by finnhlarsen** [comment]
Good idea - I think the easiest solution would be if you submitted a PR?

**Comment by RasmusFredensborg** [comment]
#3

**Comment by RasmusFredensborg** [comment]
This is fixed by #3
